DEVELOPING_OPAL/tools/src/main/scala/org/opalj/support/info/StringAnalysisReflectiveCalls.scala

```diff
@@ -0,0 +1,311 @@
+            (stmt.astID: @switch) match {
+                case Assignment.ASTID ⇒ stmt match {
+                    case Assignment(_, _, c: StaticFunctionCall[V]) ⇒
+                        processFunctionCall(ps, m, c, resultMap)
+                    case Assignment(_, _, c: VirtualFunctionCall[V]) ⇒
+                        processFunctionCall(ps, m, c, resultMap)
+                    case _ ⇒
+                }
+                case ExprStmt.ASTID ⇒ stmt match {
+                    case ExprStmt(_, c: StaticFunctionCall[V]) ⇒
+                        processFunctionCall(ps, m, c, resultMap)
+                    case ExprStmt(_, c: VirtualFunctionCall[V]) ⇒
+                        processFunctionCall(ps, m, c, resultMap)
+                    case _ ⇒
+                }
+                case _ ⇒
+            }
+        }
+    }
+
+    private def continuation(
+        ps: PropertyStore, m: Method, resultMap: ResultMapType
+    )(eps: SomeEPS): ProperPropertyComputationResult = {
+        eps match {
+            case FinalP(tac: TACAI) ⇒
+                processStatements(ps, tac.tac.get.stmts, m, resultMap)
+                Result(m, tac)
+            case InterimLUBP(lb, ub) ⇒
+                InterimResult(
+                    m, lb, ub, List(eps), continuation(ps, m, resultMap)
+                )
+            case _ ⇒ throw new IllegalStateException("should never happen!")
+        }
+    }
+
+    override def doAnalyze(
+        project: Project[URL], parameters: Seq[String], isInterrupted: () ⇒ Boolean
+    ): ReportableAnalysisResult = {
+        val manager = project.get(FPCFAnalysesManagerKey)
+        project.get(RTACallGraphKey)
+        implicit val (propertyStore, analyses) = manager.runAll(
+            LazyInterproceduralStringAnalysis
+        // LazyIntraproceduralStringAnalysis
+        )
+
+        // Stores the obtained results for each supported reflective operation
+        val resultMap: ResultMapType = mutable.Map[String, ListBuffer[StringConstancyInformation]]()
+        relevantMethodNames.foreach { resultMap(_) = ListBuffer() }
+
+        project.allMethodsWithBody.foreach { m ⇒
+            // To dramatically reduce work, quickly check if a method is relevant at all
+            if (instructionsContainRelevantMethod(m.body.get.instructions)) {
+                var tac: TACode[TACMethodParameter, DUVar[ValueInformation]] = null
+                val tacaiEOptP = propertyStore(m, TACAI.key)
+                if (tacaiEOptP.hasUBP) {
+                    if (tacaiEOptP.ub.tac.isEmpty) {
+                        // No TAC available, e.g., because the method has no body
+                        println(s"No body for method: ${m.classFile.fqn}#${m.name}")
+                    } else {
+                        tac = tacaiEOptP.ub.tac.get
+                        processStatements(propertyStore, tac.stmts, m, resultMap)
+                    }
+                } else {
+                    InterimResult(
```

Comment:
who is processing this result? seems to be unused.

Reply:
Removed in https://github.com/opalj/opal/pull/1/commits/c407442a4ce07fbcdbdf7032e6ceade310d4aaed